src/App.js

```diff
@@ -1,12 +1,27 @@
 import React from 'react';
+import { BrowserRouter as Router, Route, Link } from 'react-router-dom';
+import Index from './components/pages/Index';
+import Login from './components/pages/Login';
 
 function App() {
   return (
-    <div className="App">
-      <header>
-        <h1>Tieme Ndo</h1>
-      </header>
-    </div>
+    <Router>
+      <div className="App">
+        <nav>
+          <ul>
+            <li>
+              <Link to="/">Home</Link>
+            </li>
+            <li>
+              <Link to="/login/">Login</Link>
+            </li>
+          </ul>
+        </nav>
```

Comment:
❓ Could the `<nav> ... </nav>` live in a component on its own?

Reply:
I totally agree, this will be a component in its own. My idea was to me a simple and fast placeholder just to test that the Router works with those dummy links and pages, and all of that will definitely change when we start working on the real components (nav, etc) 🤠